editor/scene_tree_dock.cpp

```diff
@@ -1061,6 +1061,28 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 				}
 			}
 		} break;
+		case TOOL_TOGGLE_SCENE_UNIQUE_NAME: {
+			List<Node *> selection = editor_selection->get_selected_node_list();
+			List<Node *>::Element *e = selection.front();
+			if (e) {
+				UndoRedo *undo_redo = &editor_data->get_undo_redo();
+				Node *node = e->get();
+				bool enabled = node->is_unique_name_in_owner();
+				if (!enabled && get_tree()->get_edited_scene_root()->get_node_or_null("%" + String(node->get_name())) != nullptr) {
```

Comment:
```suggestion
				if (!enabled && get_tree()->get_edited_scene_root()->get_node_or_null("%" + String(node->get_name()))) {
```
Explicitly comparing to null is not necessary.

Reply:
I find it adds to more readability in general as it helps differentiate from functions that return bool